src/cli-constant.js

```diff
@@ -239,13 +252,22 @@ function dedent(str) {
   return str.replace(new RegExp(`^ {${spaces}}`, "gm"), "").trim();
 }
 
+function kebabToCamel(str) {
+  return str.replace(/-([a-z])/g, (_, char) => char.toUpperCase());
+}
+
 function normalizeDetailOptions(rawDetailOptions) {
   const names = Object.keys(rawDetailOptions).sort();
 
   const normalized = names.map(name => {
     const option = rawDetailOptions[name];
     return Object.assign({}, option, {
       name,
+      formatOption:
```

Comment:
Having `category: "format"` and `formatOption: true` is a bit confusing. How about using `option.category === "format"` here and hard-coding the `filepath` exception in `getOptions()`?

Reply:
But it'll cause `--cursor-offset` to be listed in Format Options, is that okay?

Reply:
IMO, `category` is just for usage to categorize, and `formatOption` is for `getOptions` to pick options, maybe a better naming for `formatOption` would be better? I'd like to put everything in `detailOptions` if possible.

Reply:
Ah. Maybe rename to `forwardToApi`?

Reply:
👍